pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -30,7 +30,9 @@ while read pkg_spec; do
   pkg_sha256="$(nix-hash --type sha256 --flat --base32 "$(dirname "$pkg_spec")"/*.nupkg)"
 
   pkg_src="$(jq --raw-output '.source' "$(dirname "$pkg_spec")/.nupkg.metadata")"
-  if [[ $pkg_src != https://api.nuget.org/* ]] && [[ ! -d $pkg_src ]]; then
+  if [[ -d $pkg_src ]]; then
+      continue
```

Comment:
I'm assuming in the case this is a directory the dependencies are contained in the source itself, is that right?

Reply:
Yeah, the assumption here is that if it's in /nix/store, we don't need to write it.  Previously it would just write it without a url, implying that it's available on api.nuget.org, which seems wrong.

Reply:
Yeah, that doesn't make much sense to me either. Thanks for the fix.